framework/src/setroubleshoot/server.py

```diff
@@ -116,6 +116,13 @@ def sighandler(signum, frame):
     import sys    
     sys.exit()
 
+def polling_failled_handler(signum, frame):
+    log_debug("received signal=%s" % signum)
+    if signum == signal.SIGALRM:
```

Comment:
Can be the signal other than SIGALRM in this handler?

Reply:
No, It shouldn't.